docs/rules/rrd/function-size.md

```diff
@@ -8,3 +8,116 @@ Checks if functions inside `script setup` block are less than 20 lines of code.
 - **Readability**: Smaller functions are easier to read and understand.
 - **Maintainability**: Smaller functions are easier to maintain and refactor.
 - **Testability**: Smaller functions are easier to test.
+
+## 😱 Examples of code for which this rule will throw a warning
+
+::: warning
+The following code contains functions that exceed the recommended size limit. Large functions can be difficult to read, understand, and maintain.
+:::
+
+```javascript
+function dummyRegularFunction() {
+  const firstName = 'john'
+  const lastName = 'doe'
+  const age = 30
+
+  if (age < 18) {
+    console.log('Too young for this function!')
+  }
+  else {
+    console.log('Hello ', firstName)
+  }
+
+  const hobbies = ['reading', 'gaming', 'cooking']
+  for (const hobby of hobbies) {
+    console.log('I enjoy ', hobby)
+  }
+
+  const getRandomNumber = () => Math.floor(Math.random() * 100)
+  const randomNum = getRandomNumber()
+  console.log('Random number: ', randomNum)
+
+  return 'Function execution complete!'
+}
+```
+
+Another example with an arrow function:
+
+```javascript
+const getOpenBookings = page =>
+  axios
+    .get(`${import.meta.env.VITE_APP_API_URL}bookings/listOpen.json?page=${page}`, store.tokenHeader)
+    .then((res) => {
+      bookings.value = res.data.bookings
+      paginate.value = res.data.paginate
+
+      const hobbies = ['reading', 'gaming', 'cooking']
+      for (const hobby of hobbies) {
+        console.log('I enjoy ', hobby)
+      }
+
+      bookings.value = res.data.bookings
+      paginate.value = res.data.paginate
+      bookings.value = res.data.bookings
+      paginate.value = res.data.paginate
+      bookings.value = res.data.bookings
+      paginate.value = res.data.paginate
+      // Additional repetitive lines...
+    })
+    .catch(err => console.error(err))
+```
+
+## 🤩 How to fix it?
+
+::: tip
+Refactor the function to reduce its size by breaking it down into smaller, more focused functions. This improves readability, testability, and maintainability.
+:::
+
+For the first example:
+
+```javascript
+function logAgeMessage(age) {
```

Comment:
One rule will be trigger for a non arrow function if I remember correctly

Reply:
I dont get it. 

Under that regular function example, there is one with arrow function

![image](https://github.com/user-attachments/assets/ab19cc7f-6be0-45a9-bd92-2aef2021338a)

whats missing? 🤔

Reply:
my bad. I thought we have a rule against nomal functions but we do not have any.